sssd_test_framework/hosts/client.py

```diff
@@ -30,6 +30,10 @@ def __init__(self, *args, **kwargs) -> None:
 
         self._features: dict[str, bool] | None = None
 
+    def pytest_setup(self) -> None:
```

Comment:
Don't forget to run `super().pytest_setup()`

Reply:
Fixed